library/coretests/tests/cell.rs

```diff
@@ -549,3 +549,17 @@ fn refcell_swap() {
         forget((c, d));
     };
 }
+
+#[test]
```

Comment:
I believe @pvdrz has been putting these new coverage tests in `library/coretests/tests/not_doctests.rs` (which has many additions). If you would prefer them here, please annotate that they're our additions so someone doesn't just delete them in a PR rotation merge.